cognite_toolkit/_cdf_tk/utils/table_writers.py

```diff
@@ -20,7 +19,8 @@
 
 FileFormat: TypeAlias = Literal["csv", "parquet", "yaml"]
 DataType: TypeAlias = Literal["string", "integer", "float", "boolean", "json"]
-Rows: TypeAlias = list[dict[str, str | int | float | bool | datetime | date | object | list | None]]
+JsonVal: TypeAlias = None | str | int | float | bool | dict[str, "JsonVal"] | list["JsonVal"]
+Rows: TypeAlias = list[dict[str, str | int | float | bool | JsonVal | None]]
```

Comment:
I think the fact that this expands to duplicates of identical types is a bit strange, but not blocking.